lib/matplotlib/backend_bases.py

```diff
@@ -51,14 +51,12 @@
 from matplotlib.path import Path
 
 try:
-    from PIL import Image
```

Comment:
This was just a test import to find out if PIL is available. Superseeded by importing PILLOW_VERSION.